smarts/core/sumo_traffic_simulation.py

```diff
@@ -466,7 +466,11 @@ def _step(self, dt):
         # we tell SUMO to step through dt more seconds of the simulation
         self._cumulative_sim_seconds += dt
         try:
-            self._traci_conn.simulationStep(self._cumulative_sim_seconds)
+            # Suppress errors here, to avoid a known (and likely benign)
+            # error related to removing vehicles.
+            # See: https://github.com/huawei-noah/SMARTS/issues/1155
+            with suppress_output(stderr=False):
+                self._traci_conn.simulationStep(self._cumulative_sim_seconds)
```

Comment:
Is sumo outputting errors on `stdout` for some reason?

Reply:
Weird. I suppose they consider that it is not crashing, perhaps it is just reasonable to not output the sumo stdout logs because we have it set to log to a file anyway.